src/python/CRABClient/JobType/Analysis.py

```diff
@@ -158,17 +160,15 @@ def run(self, filecacheurl = None):
                         ISBSize = int(re_match.group(1))
                         ISBSizeLimit = int(re_match.group(2))
                         reason  = "%sError%s:" % (colors.RED, colors.NORMAL)
-                        reason += " Input sanbox size is ~%sMB. This is bigger than the maximum allowed size of %sMB." % (ISBSize/1024/1024, ISBSizeLimit/1024/1024)
-                        ISBContent = sorted(tb.content, reverse=True)
-                        biggestFileSize = ISBContent[0][0]
-                        ndigits = int(math.ceil(math.log(biggestFileSize+1, 10)))
-                        reason += "\nInput sanbox content sorted by size[Bytes]:"
-                        for (size, name) in ISBContent:
-                            reason += ("\n%" + str(ndigits) + "s\t%s") % (size, name)
+                        reason += " Input sandbox size is ~%sMB. This is bigger than the maximum allowed size of %sMB." % (ISBSize/1024/1024, ISBSizeLimit/1024/1024)
+                        reason += tb.printSortedContent()
                         raise ClientException(reason)
-                raise hte
+            except SandboxTooBigException as e:
```

Comment:
I guess you meant "the changes proposed above".
OK. I will try. I am not really used to exceptions. Guess I have a coding style where they are pretty much handled as exit codes.